docs/development_setup.md

```diff
@@ -47,8 +47,9 @@ A minimal database setup for running tests is provided. If the build argument `O
 
 ### Requirements
 
-- Python 3.8+
-- Poetry 1.1.0+
+- Python: 3.8 > 3.11
+- R: ^4.0
+- Poetry: ^1.0
```

Comment:
I wouldn't use `^` in a manual. I would suggest to use human readable Syntax, e.g. using `≥` or `4.x`.

Reply:
Yes, thats easier to read. Will fix it.